beanstalk-ec2-model-serving/src/main/java/com/example/djlInferenceBeanStalk/Handler.java

```diff
@@ -0,0 +1,129 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.example.djlInferenceBeanStalk;
+
+import ai.djl.ModelException;
+import ai.djl.inference.Predictor;
+import ai.djl.modality.Classifications;
+import ai.djl.modality.cv.Image;
+import ai.djl.modality.cv.ImageFactory;
+import ai.djl.modality.cv.transform.ToTensor;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.repository.zoo.Criteria;
+import ai.djl.repository.zoo.ModelZoo;
+import ai.djl.repository.zoo.ZooModel;
+import ai.djl.translate.TranslateException;
+import ai.djl.translate.Translator;
+import ai.djl.translate.TranslatorContext;
+import ai.djl.util.Utils;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.logging.Logger;
+
+@RestController
+public class Handler {
+    Logger logger = Logger.getLogger(Handler.class.getName());
+
+    private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
+
+    static {
+        // DJL saves model and native libraries in cache folder.
+        // In AWS-Lambda only /tmp folder is writable.
+        System.setProperty("DJL_CACHE_DIR", "/tmp/djl_cache");
+    }
+
+    /**
+     * This method is the Rest endpoint where the user can post their images
+     * to run inference against a model of their choice using DJL!
+     * @param inputRequest The request body containing the image in String format
+     * @return Returns the top 5 probable items from the model output
+     * @throws IOException
+     */
+    @PostMapping(value = "/inference")
+    public byte[] handleRequest(@RequestBody Request inputRequest) throws IOException {
+        // Grabs the string image from the input
+        String imageInput = inputRequest.getImageData();
+        try {
+            // Decodes the string and transforms it into an image
+            byte[] imgBytes = Base64.getDecoder().decode(imageInput);
```

Comment:
I think we should take binary image instead.

Reply:
Sure we can do either one I was just following Qing's example where he uses the string representation